src/desktop/secret.rs

```diff
@@ -19,8 +19,12 @@
 //! }
 //! ```
 
-use std::{io::Read, os::unix::prelude::AsRawFd};
+use std::os::unix::prelude::AsRawFd;
 
+#[cfg(feature = "async-std")]
+use async_std::{os::unix::net::UnixStream, prelude::*};
+#[cfg(all(feature = "tokio_runtime", not(feature = "async-std")))]
```

Comment:
i would keep feature=tokio_runtime without the async-std check & add a compile_error on lib.rs similar to when you enable both gtk3/gtk4

Reply:
Done. Added it to oo7 PR too.